pkgs/tfsec/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
```

Comment:
Doesnt this already existing in nixpkgs?

Reply:
It does, but a different version. If we allow the daily job to build images off upstream master nixpkgs, we wind up with the latest version - 0.63.1. Running that version of `tfsec` produces a LOT of findings. Whether we decide to remediate or ignore those is one thing, but definitely one that should be reserved for its own issue before deciding to upgrade. So for now I intentionally locked it to the version that the current PRs use.

Reply:
Ahhh gotcha, makes sense! In that case would you be ok with just doing an `override` inside `imgs/nwi-pr-utils/default.nix`. Checkout https://ryantm.github.io/nixpkgs/using/overrides/ section `<pkg>.overrideDerivation`. Id rather do it with an `override` so that way anyone using the nix-garage overlay doesnt inherit the pinned version of `tfsec`

Reply:
thanks for making the update to overrides :)